app/models/notifications/mentioned_in_comment.rb

```diff
@@ -11,7 +11,7 @@ def deleted_translation_key
     end
 
     def self.filter_mentions(mentions, mentionable, _recipient_user_ids)
-      mentions.joins(:person).merge(Person.allowed_to_be_mentioned_in_a_comment_to(mentionable.parent))
+      mentions.includes(:person).merge(Person.allowed_to_be_mentioned_in_a_comment_to(mentionable.parent)).distinct
```

Comment:
Why change from `joins` to `includes`? Both seem to work fine here, so `includes` should be okay. But is there anything better in using `includes`? I checked how it changes the SQL request and there seem to be nothing semantically different.